server/src/main/java/com/linecorp/centraldogma/server/internal/mirror/GitMirror.java

```diff
@@ -410,10 +428,9 @@ private Revision remoteCurrentRevision(
     }
 
     private static ObjectId fetchRemoteHeadAndGetCommitId(
-            Git git, String headBranchRefName) throws GitAPIException, IOException {
-        final FetchCommand fetch = git.fetch();
+            GitWithAuth git, String headBranchRefName) throws GitAPIException, IOException {
+        final FetchCommand fetch = git.fetch(1);
         final FetchResult fetchResult = fetch.setRefSpecs(new RefSpec(headBranchRefName))
-                                             .setCheckFetchedObjects(true)
```

Comment:
I guess not too important, but is there a reasoning for not validating fetched objects anymore?